### PR TITLE
Add GetBlock Tutorial to the documentation section

### PR DESCRIPTION
**Pull request by JoGetBlock**
Here's a tutorial on how to create a smart contract on Polygon using GetBlock (an RPC provider supporting Polygon Blockchain)

### CODE REVIEW

**Comment by ashucoder9** [comment]
Hey @JoGetBlock, this PR has some errors that need fixing before we proceed with a review. I am listing them down:
1. Incorrect JSX tags causing build failures, probably missing files too
2. Update sidebars.js file to get your document shown on the Wiki

I recommend installing the setup on your local machine and checking if everything works fine. Otherwise, let us know in the comment, and we'll help fix the issues and publish this document.

**Review by ashucoder9 — CHANGES_REQUESTED** [review]
Please try to include the image files instead of referencing their cloud links. Also, add the file to sidebars.js for display.

**Comment by cerberushades** [comment]
@JoGetBlock Hey, are you still interested in adding your tutorials to our wiki?

**Comment by JoGetBlock** [comment]
> @JoGetBlock Hey, are you still interested in adding your tutorials to our wiki?

Sorry, I've been away, yes, we're still interested
Let me fix everything with the tech team, thank you

**Review by JoGetBlock — COMMENTED** [review]
Please review the changes

**Comment by ashucoder9** [comment]
Hey @Ishmaello @JoGetBlock, I've fixed the issues and content to a good extent. But still, need your input on the below points:

- GetBlock is mainly an RPC provider which can be added to Metamask before deploying contracts. Is it necessary to deploy a sample contract? Or can we replace that with a single paragraph outlining that you may go to Remix now and deploy your contract? In my opinion, we don't need another section for that.
- The image quality is poor, would be great if you can update that.

File: docs/develop/getblock.md [hunk 1]
```diff
@@ -49,15 +49,15 @@ Go to the [official Remix](https://remix.ethereum.org/) website. Remix is a Web-
 
 1. Let's take `Storage.sol` from Remix as our example of a smart contract.
 
-  ![img](https://i.ibb.co/7JxRcS5/unnamed-2.png)
+  ![img](https://drive.google.com/file/d/12PEfBR7Tju9ReVYVtipSn3iR_rndMxEs/view?usp=sharing)
 
 2. Deploy the smart contract.
 
-  ![img](https://i.ibb.co/FHQvbb0/unnamed-3.png)
```
Comment (Ishmaello) [inline docs/develop/getblock.md]: I appreciate for all your good work I'm becoming better understanding I even con

**Comment by JoGetBlock** [comment]
> Hey @Ishmaello @JoGetBlock, I've fixed the issues and content to a good extent. But still, need your input on the below points:
> 
> * GetBlock is mainly an RPC provider which can be added to Metamask before deploying contracts. Is it necessary to deploy a sample contract? Or can we replace that with a single paragraph outlining that you may go to Remix now and deploy your contract? In my opinion, we don't need another section for that.
> * The image quality is poor, would be great if you can update that.

@ashucoder9 please, take a look at the changes

**Review by JoGetBlock — COMMENTED** [review]
Please review the latest changes

**Review by JoGetBlock — COMMENTED** [review]
@DannyS03 also let me know if there are any other editions here, thank you Danny!

**Comment by JoGetBlock** [comment]
> Hey @JoGetBlock sorry for being so strict here, but we're striving to make the best experience for our docs users. So, to improve the article visuals, I'd centralize the screenshots and try to follow some common ground for their size as well. I'd also avoid 1 image coming after another one without any text in between. Finally, I'd suggest that you take a look at the other tutorials in the same section.

It's fine, thank you for helping, it's just that I've never done these before, that's why I simply followed the requirements sent from the Polygon team, will make the editions and come back.

File: docs/develop/getblock.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+Following the tutorial, you will:
+1. Get started on GetBlock and access GetBlock test nodes
+2. Create a wallet address with Metamask
+3. Add balance to the wallet (using test tokens)
+4. Use Remix to compile and deploy a Smart Contract
+5. Check contract status
+
+### Preparatory Steps
+
+1. The first step is to [sign up with Getblock](https://account.getblock.io/sign-in) via wallet or email
```
Comment (DannyS03) [inline docs/develop/getblock.md:32]:
```suggestion
1. Sign up with Getblock by visiting the [official website](https://account.getblock.io/sign-in) and creating an account using your wallet or email.
```

File: docs/develop/getblock.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/guide1.png)
+
+2. Create an endpoint for Mumbai Testnet
```
Comment (DannyS03) [inline docs/develop/getblock.md:36]: I'd create a list that includes step-by-step instructions, such as:

```
Create an endpoint for the Mumbai Testnet on Getblock by following these steps:

- Go to the Getblock website and log in to your account.
- Click on "API Keys" in the left-hand menu.
- Click on "Create API Key" and select "Ethereum" as the blockchain.
- Choose "Mumbai Testnet" as the network and give your endpoint a name.
- Click "Create."
- Once your endpoint is created, copy the RPC URL.
```

File: docs/develop/getblock.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/guide2.png)
+
+  Then enter Metamask and choose to connect a network manually, connect your Metamask to the Mumbai Testnet adding your GetBlock endpoint in the **RPC URL** line
```
Comment (DannyS03) [inline docs/develop/getblock.md:40]: Same idea here, such as:

```
Connect Metamask to the Mumbai Testnet by following these steps:

- Open Metamask and click on the network dropdown.
- Click on "Custom RPC" at the bottom of the list.
- In the "RPC URL" field, paste the URL of your Getblock endpoint.
- Click "Save."

```

File: docs/develop/getblock.md [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/guide3.png)
+  ![img](https://storage.getblock.io/web/blog/article-images/guide4.png)
+
+If you have zero balance in your account, you can request tokens from the [Mumbai faucet](/develop/tools/matic-faucet.md).
```
Comment (DannyS03) [inline docs/develop/getblock.md:45]:
```suggestion
If you don't have any tokens in your account, you can request some from the Mumbai faucet through the [faucet website](https://chat.openai.com/develop/tools/matic-faucet.md).
```

File: docs/develop/getblock.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+
+## Smart Contract Development
+
+Go to the [official Remix](https://remix.ethereum.org/) website. Remix is a Web-based IDE for Ethereum smart contract development in Solidity.
```
Comment (DannyS03) [inline docs/develop/getblock.md:49]:
```suggestion
1. Go to the [official Remix](https://remix.ethereum.org/) website, which is a Web-based IDE for Ethereum smart contract development in Solidity.
```

File: docs/develop/getblock.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  ![img](https://storage.getblock.io/web/blog/article-images/guide3.png)
+  ![img](https://storage.getblock.io/web/blog/article-images/guide4.png)
+
+If you have zero balance in your account, you can request tokens from the [Mumbai faucet](/develop/tools/matic-faucet.md).
+
+## Smart Contract Development
+
+Go to the [official Remix](https://remix.ethereum.org/) website. Remix is a Web-based IDE for Ethereum smart contract development in Solidity. 
+
+1. Let's take `Storage.sol` from Remix as our example of a smart contract.
```
Comment (DannyS03) [inline docs/develop/getblock.md:51]:
```suggestion
2. Open the "Storage.sol" file in Remix, which we will use as an example for our smart contract.
```

File: docs/develop/getblock.md [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/guide5.png)
+
+2. Deploy the smart contract.
```
Comment (DannyS03) [inline docs/develop/getblock.md:55]:
```suggestion
3. Deploy the smart contract by clicking on the "Deploy & run transactions" icon in the left-hand menu.
```

File: docs/develop/getblock.md [hunk 9]
```diff
@@ -0,0 +1,93 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/guide6.png)
+
+3. Choose the Injected provider as **Metamask**.
```
Comment (DannyS03) [inline docs/develop/getblock.md:59]:
```suggestion
4. Choose "Injected provider" as the provider and select "Metamask" as the wallet.
```

File: docs/develop/getblock.md [hunk 10]
```diff
@@ -0,0 +1,93 @@
...
+
+2. Deploy the smart contract.
+
+  ![img](https://storage.getblock.io/web/blog/article-images/guide6.png)
+
+3. Choose the Injected provider as **Metamask**. 
+
+ ![img](https://storage.getblock.io/web/blog/article-images/guide7.png)
+  
+  Check that our smart contract has been created on the network.
```
Comment (DannyS03) [inline docs/develop/getblock.md:63]:
```suggestion
5. Check that your smart contract has been created on the network by looking at the Remix output panel.
```

File: docs/develop/getblock.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+  
+ ![img](https://storage.getblock.io/web/blog/article-images/guide8.png)
+
+Smart contract URL: https://mumbai.polygonscan.com/address/0xa4d552dce0e26564ec1737638705584ce9ed351b
+
+All done! Congratulations on deploying your smart contract.
```
Comment (DannyS03) [inline docs/develop/getblock.md:69]:
```suggestion
Congratulations, you have successfully deployed a smart contract on the Mumbai Testnet! 
```

File: docs/develop/getblock.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
...
+
+## Testing and Verification
+
+Once you have deployed your smart contract on the Polygon test network using GetBlock, it is important to test and verify its functionality to ensure that it works as intended. Here are some steps you can take for testing and verification:
+
```
Comment (DannyS03) [inline docs/develop/getblock.md:74]: Would also suggest transforming this section into more explicit steps, starting with something along the lines of:

```
1. Open the smart contract in Remix and locate the storage variable you want to test. In this example, we will use a variable called "storage" that initially has a value of 12."
```

Striving to create clear + direct and actionable steps.

```
2. Change the value of the variable to 14 by entering "14" in the "Store" field and saving the value on the network after confirming the operation in Metamask.
```

And so on.

**Review by JoGetBlock — COMMENTED** [review]
The required changes added, please check now.

**Comment by DannyS03** [comment]
Thank you, @JoGetBlock, for incorporating the changes. Could you please confirm if you have resolved all the suggestions, and if not, could you kindly point out which ones still need to be addressed?

**Comment by JoGetBlock** [comment]
> 

Hi Danny, I've tried to accept and add all the changes you've asked for (the text corrections, more explicit explanations, the image formats), so I'd confirm that

File: docs/develop/getblock.md [hunk 13]
```diff
@@ -0,0 +1,112 @@
...
+Smart contract URL: https://mumbai.polygonscan.com/address/0xa4d552dce0e26564ec1737638705584ce9ed351b
+
+If your smart contract has been successfully deployed, you should see the contract details including its address, balance, and transaction history. Congratulations, you have successfully deployed a smart contract on the Mumbai Testnet! 
+
+## Testing and Verification
+
+- Once you have deployed your smart contract on the Polygon test network using GetBlock, it is important to test and verify its functionality to ensure that it works as intended. Here are some steps you can take for testing and verification:
+
+- Open the smart contract in Remix and locate the storage variable you want to test. In this example, we will use a variable called "storage" that initially has a value of 12.
```
Comment (DannyS03) [inline docs/develop/getblock.md:92]:
```suggestion
1. Open the smart contract in Remix and locate the storage variable you want to test. In this example, we will use a variable called "storage" that initially has a value of 12."
```

File: docs/develop/getblock.md [hunk 14]
```diff
@@ -0,0 +1,112 @@
...
+
+## Testing and Verification
+
+- Once you have deployed your smart contract on the Polygon test network using GetBlock, it is important to test and verify its functionality to ensure that it works as intended. Here are some steps you can take for testing and verification:
+
+- Open the smart contract in Remix and locate the storage variable you want to test. In this example, we will use a variable called "storage" that initially has a value of 12. 
+
+ ![img](https://storage.getblock.io/web/blog/article-images/img7+(1).png)
+
+- Find the "Store" field and enter the number "14". This will change the value of the "storage" variable from 12 to 14.
```
Comment (DannyS03) [inline docs/develop/getblock.md:96]:
```suggestion
2. Change the value of the variable to 14 by entering "14" in the "Store" field and saving the value on the network after confirming the operation in Metamask.
```

File: docs/develop/getblock.md [hunk 15]
```diff
@@ -0,0 +1,112 @@
...
+
+- Click on the "Transact" button to confirm the transaction in Metamask.
```
Comment (DannyS03) [inline docs/develop/getblock.md:98]:
```suggestion
3. Click on the "Transact" button to confirm the transaction in Metamask.
```

File: docs/develop/getblock.md [hunk 16]
```diff
@@ -0,0 +1,112 @@
...
+
+ ![img](https://storage.getblock.io/web/blog/article-images/imga10.png)
+
+- Wait for the transaction to be added to the network block. Once the transaction is confirmed, go to the "Read Contract" section under "Deployed Contracts".
```
Comment (DannyS03) [inline docs/develop/getblock.md:102]:
```suggestion
4. Wait for the transaction to be added to a network block. Once the transaction is confirmed, navigate to the "Read Contract" section under "Deployed Contracts".
```

File: docs/develop/getblock.md [hunk 17]
```diff
@@ -0,0 +1,112 @@
...
+
+- Find the "storage" variable and click on the "Read" button. The new value of the variable should now be displayed as 14.
```
Comment (DannyS03) [inline docs/develop/getblock.md:104]:
```suggestion
4. Find the "storage" variable and click on the "Read" button. The new value of the variable should now be displayed as "14".
```

File: docs/develop/getblock.md [hunk 18]
```diff
@@ -0,0 +1,112 @@
...
+
+3. Then enter Metamask and choose to connect a network manually, connect your Metamask to the Mumbai Testnet adding your GetBlock endpoint in the **RPC URL** line
+
+  ![img](https://storage.getblock.io/web/blog/article-images/imga3.png)
+
+If you don't have any tokens in your account, you can request some from the Mumbai faucet through the [faucet website](https://chat.openai.com/develop/tools/matic-faucet.md).
+
+## Smart Contract Development
+
+- Go to the [official Remix](https://remix.ethereum.org/) website. Remix is a Web-based IDE for Ethereum smart contract development in Solidity.
```
Comment (DannyS03) [inline docs/develop/getblock.md:49]:
```suggestion
1. Visit the [official Remix website](https://remix.ethereum.org/). Remix is a Web-based IDE for Ethereum smart contract development in Solidity. 
```

File: docs/develop/getblock.md [hunk 19]
```diff
@@ -0,0 +1,112 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/imga3.png)
+
+If you don't have any tokens in your account, you can request some from the Mumbai faucet through the [faucet website](https://chat.openai.com/develop/tools/matic-faucet.md).
+
+## Smart Contract Development
+
+- Go to the [official Remix](https://remix.ethereum.org/) website. Remix is a Web-based IDE for Ethereum smart contract development in Solidity. 
+
+- Click on the "Create a new file" button located on the left side of the page.
```
Comment (DannyS03) [inline docs/develop/getblock.md:51]:
```suggestion
2. Click on the "Create a new file" button located on the left side of the page.
```

File: docs/develop/getblock.md [hunk 20]
```diff
@@ -0,0 +1,112 @@
...
+
+- Select “Storage.sol” from Remix as an example of a contract.
```
Comment (DannyS03) [inline docs/develop/getblock.md:53]:
```suggestion
3. Select “Storage.sol” from Remix as an example of a contract.
```

File: docs/develop/getblock.md [hunk 21]
```diff
@@ -0,0 +1,112 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/imga4.png)
+
+- Proceed by deploying the smart contract by clicking on the “Deploy & Run Transactions” tab.
```
Comment (DannyS03) [inline docs/develop/getblock.md:57]:
```suggestion
4. Deploy the smart contract by clicking on the “Deploy & Run Transactions” tab.
```

File: docs/develop/getblock.md [hunk 22]
```diff
@@ -0,0 +1,112 @@
...
+
+- In Remix, click on the “Deploy & Run Transactions” tab located on the left-hand side of the screen.
```
Comment (DannyS03) [inline docs/develop/getblock.md:59]:
```suggestion
5. Click on the “Deploy & Run Transactions” tab in Remix, located on the left-hand side of the screen. 
```

File: docs/develop/getblock.md [hunk 23]
```diff
@@ -0,0 +1,112 @@
...
+
+- Select “Storage.sol” from Remix as an example of a contract.
+
+  ![img](https://storage.getblock.io/web/blog/article-images/imga4.png)
+
+- Proceed by deploying the smart contract by clicking on the “Deploy & Run Transactions” tab.
+
+- In Remix, click on the “Deploy & Run Transactions” tab located on the left-hand side of the screen. 
+
+- In the “Contract” dropdown menu, select the name of the smart contract you want to deploy and click on the “Deploy” button to deploy the smart contract.
```
Comment (DannyS03) [inline docs/develop/getblock.md:61]:
```suggestion
6. In the “Contract” dropdown menu, select the name of the smart contract you want to deploy and click on the “Deploy” button to deploy the smart contract.
```

File: docs/develop/getblock.md [hunk 24]
```diff
@@ -0,0 +1,112 @@
...
+A Metamask window will pop up asking you to confirm the transaction. Click on the “Confirm” button to proceed.Wait for the transaction to be confirmed on the Polygon network. 
+
+- You can track the status of the transaction in the Metamask window or by checking the transaction hash on the Polygon explorer.
```
Comment (DannyS03) [inline docs/develop/getblock.md:64]:
```suggestion
> You can track the status of the transaction in the Metamask window or by checking the transaction hash on the Polygon explorer.
```

File: docs/develop/getblock.md [hunk 25]
```diff
@@ -0,0 +1,112 @@
...
+
+- Once the transaction has been confirmed, you should see a green checkmark and a message indicating that the contract has been deployed successfully.
```
Comment (DannyS03) [inline docs/develop/getblock.md:66]:
```suggestion
Once the transaction has been confirmed, you should see a green checkmark and a message indicating that the contract has been deployed successfully.
```

File: docs/develop/getblock.md [hunk 26]
```diff
@@ -0,0 +1,112 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/imga5.png)
+
+Locate the storage variable you want to test.
```
Comment (DannyS03) [inline docs/develop/getblock.md:70]:
```suggestion
7. Locate the storage variable you want to test.
```

File: docs/develop/getblock.md [hunk 27]
```diff
@@ -0,0 +1,112 @@
...
+
+- Choose "Injected provider" as the provider and select "Metamask" as the wallet.
```
Comment (DannyS03) [inline docs/develop/getblock.md:72]:
```suggestion
8. Choose "Injected provider" as the provider and select "Metamask" as the wallet. 
```

File: docs/develop/getblock.md [hunk 28]
```diff
@@ -0,0 +1,112 @@
...
+
+  ![img](https://storage.getblock.io/web/blog/article-images/imga5.png)
+
+Locate the storage variable you want to test.
+
+- Choose "Injected provider" as the provider and select "Metamask" as the wallet. 
+
+ ![img](https://storage.getblock.io/web/blog/article-images/imga6.png)
+  
+- Check that your smart contract has been created on the network by looking at the Remix output panel.
```
Comment (DannyS03) [inline docs/develop/getblock.md:76]:
```suggestion
> Check that your smart contract has been created on the network by looking at the Remix output panel.
```

File: docs/develop/getblock.md [hunk 29]
```diff
@@ -0,0 +1,112 @@
...
+
+- Click on the "Polygon (Matic) Mainnet" dropdown menu located at the top right corner of the page and select "Mumbai Testnet" from the list of available networks. Then enter the address of your smart contract in the search bar located at the top of the page.
```
Comment (DannyS03) [inline docs/develop/getblock.md:78]:
```suggestion
9. Click on the "Polygon (Matic) Mainnet" dropdown menu located at the top right corner of the page and select "Mumbai Testnet" from the list of available networks. Then enter the address of your smart contract in the search bar located at the top of the page.
```

File: docs/develop/getblock.md [hunk 30]
```diff
@@ -0,0 +1,112 @@
...
+
+- Click on the search button or hit enter.
```
Comment (DannyS03) [inline docs/develop/getblock.md:80]:
```suggestion
10. Click on the search button or hit enter.
```

**Pull request merged**